packaging/windows/sharedframework/registrykeys.wxs

```diff
@@ -13,6 +13,26 @@
           <RegistryValue Action="write" Name="1.0.0-rc2" Type="integer" Value="1" KeyPath="yes"/>
         </RegistryKey>
       </Component>
+
+<!-- Set registry keys to allow WER to genereate correct dumps-->
+      <Component Directory="TARGETDIR">
+        <RegistryKey Root="HKLM" Key="SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules">
+          <RegistryValue Action="write" Name="[DOTNETHOME]shared\$(var.FrameworkName)\$(var.FrameworkDisplayVersion)\mscordaccore.dll" Type="integer" Value="0" KeyPath="yes"/>
```

Comment:
What does $(var.FrameworkDisplayVersion) resolve to? Is it the 2 part or 3 part version?